Libplanet.Tests/Blocks/PreEvaluationBlockHeaderTest.cs

```diff
@@ -62,7 +62,30 @@ public void Constructors()
                 0,
                 0,
                 _contents.GenesisHash,
-                new[] { voteA, voteB, voteC }.ToImmutableArray());
+                new[]
+                {
+                    new VoteMetadata(
+                        0,
+                        0,
+                        _contents.GenesisHash,
+                        DateTimeOffset.UtcNow,
+                        validatorA.PublicKey,
+                        VoteFlag.Commit).Sign(validatorA),
+                    new VoteMetadata(
+                        0,
+                        0,
+                        _contents.GenesisHash,
+                        DateTimeOffset.UtcNow,
+                        validatorB.PublicKey,
+                        VoteFlag.Commit).Sign(validatorB),
+                    new VoteMetadata(
+                        0,
+                        0,
+                        _contents.GenesisHash,
+                        DateTimeOffset.UtcNow,
+                        validatorC.PublicKey,
+                        VoteFlag.Commit).Sign(validatorC),
+                }.ToImmutableArray());
```

Comment:
This might be misleading about having `BlockCommit` for the genesis block. Maybe we should add a comment about it or so?

Reply:
Sure can do, might as well polish the test a bit more then.

Reply:
Done. See aef376dcc8fad67b1679ee746c354d80733534ca.